modules/cnf-tuning-nodes-for-low-latency-via-performanceprofile.adoc

```diff
@@ -52,13 +52,78 @@ spec:
 <1> Valid values are `true` or `false`. Setting the `true` value installs the real-time kernel on the node.
 <2> Use this field to configure the topology manager policy. Valid values are `none` (default), `best-effort`, `restricted`, and `single-numa-node`. For more information, see link:https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policies[Topology Manager Policies].
 
-[id="cnf-partitioning-the-cpus_{context}"]
-== Partitioning the CPUs
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="cnf-cpu-infra-container_{context}"]
+= Restricting CPUs for infra and application containers
```

Comment:
You can't put a second H1 in a module. You need to split this module into three parts: the original content, and the concept/procedure split from #37432

Reply:
Updated as suggested.